qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -3,8 +3,9 @@
     guest_scripts = VirtIoChannel_guest_send_receive.py;windows_support.py
     guest_script_folder = /var/tmp/
     file_transfer_serial_port = vc1
-    virtio_ports = vc1 vc2
-    virtio_port_type = serialport
+    serials += " vc1 vc2"
```

Comment:
`vs` ? (just keep consistent,  they all work)

Reply:
@yanan-fu Let's keep it in this patch, this script does not work anyway, I am rewriting it.